src/witness/svcomp.ml

```diff
@@ -92,6 +96,7 @@ struct
     | False (Some spec) ->
       let result_spec = match spec with
         | UnreachCall _ -> "unreach-call"
+        | NoOverflow -> "overflow"
```

Comment:
This string needs to be `no-overflow` corresponding to the SV-COMP property name. Although it won't really matter right now if we never output `false`.

Reply:
done